src/app/project/project.component.ts

```diff
@@ -316,6 +317,23 @@ export class ProjectComponent implements OnInit, OnDestroy, AfterViewInit {
     });
   }
 
+  public learnMore() {
+    this.ngbModal = this.modalService.open(BecomeAMemberComponent, { backdrop: 'static', size: 'lg' });
+    // set input parameter
+    (<BecomeAMemberComponent>this.ngbModal.componentInstance).project = this.project;
+    // check result
+    this.ngbModal.result.then(
```

Comment:
It doesn't appear this the result is being handled in a way other than a log? Should something else be happening in this function?

Reply:
The modal encapsulates and handles all the code required to submit a user to a CAC on a project, so the logging really is the only thing going on.  If the user cancels, it does nothing, and if it submits successfully,